### PR TITLE
Fix VB outlining banner text to not break onto multiple lines

### LINKED ISSUE #6927: Outlining: Need to handle newlines in type parameter lists of parameter types better
image

### PR DESCRIPTION
**Pull request by DustinCampbell**
Fixes #6927

There's a functional change from what we had in RTM in that we will now show ByVal in the banner text if the user has it in their code. This is a very small detail though and we didn't ever remove ByVal here in Dev12.

Tagging @dotnet/roslyn-ide

### CODE REVIEW

File: src/EditorFeatures/VisualBasic/Outlining/Outliners/AccessorDeclarationOutliner.vb [hunk 1]
```diff
@@ -34,7 +17,7 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.Outlining
                     spans.Add(
                     VisualBasicOutliningHelpers.CreateRegionFromBlock(
                         methodBlock,
-                        GetBannerText(accessorDeclaration),
+                        accessorDeclaration.ConvertToSingleLine().ToString() & " " & Ellipsis,
```
Comment (CyrusNajmabadi) [inline src/EditorFeatures/VisualBasic/Outlining/Outliners/AccessorDeclarationOutliner.vb]: Would like a helper that does this work so it isn't repeated across all outliners.

**Comment by CyrusNajmabadi** [comment]
:+1:

**Comment by Pilchie** [comment]
:+1:  Love to see all that red.

File: src/EditorFeatures/VisualBasic/Outlining/Outliners/AccessorDeclarationOutliner.vb (hunk 1 above)
Comment (DustinCampbell) [inline src/EditorFeatures/VisualBasic/Outlining/Outliners/AccessorDeclarationOutliner.vb]: Added. I also went through and generally cleaned up the code in the outliners. It's quite a bit smaller now.

**Comment by CyrusNajmabadi** [comment]
:-1: not enough red.

**Comment by DustinCampbell** [comment]
:tongue:

**Comment by DustinCampbell** [comment]
retesting due to #7334

**Comment by DustinCampbell** [comment]
retest prtest/win/dbg/unit64 please

**Pull request merged**